src/scale/Time.ts

```diff
@@ -628,7 +628,7 @@ function getIntervalTicks(
     }
 
     const levelsTicksInExtent = filter(map(levelsTicks, levelTicks => {
-        return filter(levelTicks, tick => tick.value >= extent[0] && tick.value <= extent[1] && !tick.notAdd);
+        return filter(levelTicks, tick => tick.value > extent[0] && tick.value < extent[1] && !tick.notAdd);
```

Comment:
This is probably not the correct place to fix the problem. For example, the min/max labels are always hidden after this changing. You need to make sure the test cases in `timeScale-formatter.html` still works the same as before changing.

Reply:
> This is probably not the correct place to fix the problem. For example, the min/max labels are always hidden after this changing. You need to make sure the test cases in `timeScale-formatter.html` still works the same as before changing.

min/max labels work normal afeer this changing, could you check it again? Or give me a specific example ?